Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs

```diff
@@ -4,24 +4,21 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System;
-using System.Collections.Generic;
-using System.Data.Entity;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Reorder using statements to get System.* on top and the rest of the libraries in alphabetic order.